extensions/arc/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "arc",
+  "displayName": "Azure Arc",
+  "description": "%arc.description%",
+  "version": "1.0.0",
+  "publisher": "Microsoft",
+  "preview": true,
+  "license": "https://raw.githubusercontent.com/Microsoft/azuredatastudio/master/LICENSE.txt",
+  "icon": "images/extension.png",
+  "engines": {
+    "vscode": "*",
+    "azdata": "*"
+  },
+  "activationEvents": [
+    "*"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Microsoft/azuredatastudio.git"
+  },
+  "main": "./out/extension",
+  "contributes": {
+    "commands": [
+      {
+        "command": "arc.manageMiaa",
```

Comment:
Disable these in the command palette for merging into master

e.g. 
https://github.com/microsoft/azuredatastudio/blob/master/extensions/big-data-cluster/package.json#L41